feeds/feeds.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: default
+title: Feeds
+nav_order: 3
+has_children: true
+---
+
+# Central Feed Management
+
+Package sources are currently added to NuGet via NuGet.config files. This is highly configurable, but it can create challenges for restoring large and complex solutions. For example, a NuGet.config file stored on a developer's computer within their user directory may cause their development environment to behave differently from other developers' or from CI/CD environments.
+
+By configuring feeds centrally, package sources can be managed for security and repeatable builds.
+
+#### Directory.Packages.props
+```xml
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+    <ItemGroup>
+        <PackageSource key="NuGet.org" Feed="https://nuget.org/api/v3/" />
+        <PackageSource key="Local Sources" Feed="c:\installed_packages" />
```

Comment:
Is it possible to have a flag to allow disabling the cache completly on a `PackageSource` ?
Why ?
this example is perfect, when you build your own nuget and try to consume is on a project, you don't always bump the version, so you always generate `20.0.0`
Usually you just do
* code change
* build 
* pack on build enabled
* output package path go to `c:\installed_packages`
* you run `dotnet restore` else where

As of today the this will not bump the version
then `dotnet restore` will mostly `no-op` and use older build

Reply:
It's a good use case, but IMO it's out of scope of this spec.
FWIW I solve this in _either_ of two ways, usually:
1. blow away the `%userprofile%\.nuget\packages\mypackage\1.0.0` folder each time I want to restore a new version.
1. ensure the package version changes each time, using [nerdbank.gitversioning](https://github.com/dotnet/nerdbank.gitversioning) and making sure each thing I test is a new commit (whether novel or amended) so that the commit ID changes. NB.GV stamps each package with a version that includes the commit ID when working in topic branches to guarantee uniqueness.